commercetools-models/src/test/java/io/sphere/sdk/categories/commands/CategoryDeleteCommandIntegrationTest.java

```diff
@@ -30,4 +31,26 @@ public void referenceExpansion() throws Exception {
         assertThat(deletedCategory.getParent().getObj()).isNotNull().isEqualTo(parent);
         client().executeBlocking(CategoryDeleteCommand.of(parent));
     }
+
+    @Test
+    public void executionForDeletionByKey() throws Exception {
+        final String key = randomKey();
+        final CategoryDraft draft = CategoryDraftBuilder.of(randomSlug(), randomSlug()).key(key).build();
```

Comment:
How about replacing the first `randomSlug()` call to a new method `SphereTestUtils.randomLocalizedString()`?